Firestore/Source/Local/FSTQueryCache.h

```diff
@@ -98,6 +98,18 @@ NS_ASSUME_NONNULL_BEGIN
 /** Removes the cached entry for the given query data (no-op if no entry exists). */
 - (void)removeQueryData:(FSTQueryData *)queryData group:(FSTWriteGroup *)group;
 
+- (void)enumerateSequenceNumbersUsingBlock:(void (^)(FSTListenSequenceNumber sequenceNumber,
+                                                     BOOL *stop))block;
+
+- (NSUInteger)removeQueriesThroughSequenceNumber:(FSTListenSequenceNumber)sequenceNumber
```

Comment:
Comments. Removes queries with sequence numbers up to and including the given sequence number.

What does live queries have to do with it? Are they excluded? Does encountering a live query terminate the removal?